libcnb-data/src/exec_d.rs

```diff
@@ -1,5 +1,111 @@
+use crate::newtypes::libcnb_newtype;
 use serde::Serialize;
 use std::collections::HashMap;
 
-#[derive(Serialize)]
-pub type ExecD = HashMap<String, String>;
```

Comment:
Could you mention in the PR description that the reason this isn't a breaking change, is that `ExecD` was never compiled/available originally?